kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue

```diff
@@ -105,55 +128,55 @@
   import { PageNames } from '../../constants';
   import { validateUsername } from 'kolibri.utils.validators';
   import kButton from 'kolibri.coreVue.components.kButton';
-  import uiAlert from 'keen-ui/src/UiAlert';
+  import kCheckbox from 'kolibri.coreVue.components.kCheckbox';
+  import coreModal from 'kolibri.coreVue.components.coreModal';
   import kTextbox from 'kolibri.coreVue.components.kTextbox';
   import uiToolbar from 'keen-ui/src/UiToolbar';
   import logo from 'kolibri.coreVue.components.logo';
-  import uiIcon from 'keen-ui/src/UiIcon';
```

Comment:
Missing this `uiIcon` import.

Reply:
We don't need it, since we're going to be using the vf logo

Reply:
Well then leave it here, and remove it once styling is merged in?

Reply:
Wouldn't it make more sense to remove it in the style branch, then?

Reply:
Removed this line from the commit, hard reset if you locally cloned!